code/game/objects/items/rcd/RHD.dm

```diff
@@ -48,14 +48,18 @@
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 	if(upgrade & RCD_UPGRADE_SILO_LINK)
-		silo_mats = AddComponent(/datum/component/remote_materials, mapload, FALSE)
+		silo_mats = AddComponent( \
+			/datum/component/remote_materials, \
+			mapload, \
+			FALSE \
+		)
```

Comment:
Why are you doing this? Did someone tell you to do this? I get it for large procs, but this is pretty small

Reply:
:)